pkg/kubelet/kubelet.go

```diff
@@ -884,6 +891,19 @@ func (kl *Kubelet) initialNodeStatus() (*api.Node, error) {
 			Unschedulable: !kl.registerSchedulable,
 		},
 	}
+
+	if len(kl.nodeLabels) > 0 {
+		labels, err := kl.retrieveNodeLabels(kl.nodeLabels)
+		if err != nil {
+			return nil, fmt.Errorf("failed to apply the node labels, %s", err)
```

Comment:
should probably change this to reference --node-label in the error message as not to cause any confusion